arviz/plots/compareplot.py

```diff
@@ -4,8 +4,16 @@
 from .plot_utils import _scale_fig_size
 
 
-def plot_compare(comp_df, insample_dev=True, plot_standard_error=True, plot_ic_diff=True,
-                 figsize=None, textsize=None, plot_kwargs=None, ax=None):
+def plot_compare(
```

Comment:
I personally dislike this formatting, I particularly dislike the placement of the closing parentheses

Reply:
It will grow on you.  When reading this code, it should be quite easy to see what is going on.

Reply:
me too, but probably @ahartikainen is right and this is something good once we get used to it.